index.html

```diff
@@ -23,25 +23,162 @@
+         <li><a href="#"><img src="images/hand.svg" alt="hand" id="hand" class="icon"></a></li>
+        <li><a href="#"><img src="images/git.svg" alt="github_logo" id="github" class="icon"></a></li>
+        <li><a href="#"><img src="images/M.svg" alt="capital_m" id="m" class="icon"></a></li>
+    </ul>
+
+    </section>
+
+    <section  class="card" id="tonic">
+
+        <img src="images/portfolio.png" alt="a_screentshot" id="portfolio">
+
+        <h1 class="title tonic">Tonic</h1>
+
+        <img src="images/fame2.png" alt="frame" id="frame">
+
+        <p class="tonic-p">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="code-links">
+            <li class="tag">
+                <a href="" alt="">HTML</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">CSS</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">JavaScript</a>
+            </li>
         </ul>
 
+        <h3 class="button"><a href="" id="see_project">See Project</a></h3>
+
     </section>
+
+    <section id="multi" class="card">
+
+        <img src="images/portfolio2.png" alt="a_screentshot" id="portfolio">
+
+        <h1 class="title tonic">Multi-Post Stories</h1>
+        <img src="images/fame2.png" alt="frame" id="frame">
+        <p class="tonic-p">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        
+        <ul class="code-links">
+            <li class="tag">
+                <a href="" alt="">HTML</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">CSS</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">JavaScript</a>
+            </li>
+        </ul>
+
+        <h3 class="button"><a href="" id="linkp">See Project</a></h3>
+
+    </section>
+
+    <section id="tonic" class="card">
+
+        <img src="images/portfolio3.png" alt="a_screentshot" id="portfolio">
+
+        <h1 class="title tonic">Tonic</h1>
+        <img src="images/fame2.png" alt="frame" id="frame">
+        <p class="tonic-p">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        
+        <ul class="code-links">
+            <li class="tag">
+                <a href="" alt="">HTML</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">CSS</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">JavaScript</a>
+            </li>
+        </ul>
+
+        <h3 class="button"><a href="" id="linkp">See Project</a></h3>
+
+    </section>
+
+    <section id="multi2" class="card">
+
+        <img src="images/portfolio4.png" alt="a_screentshot" id="portfolio">
+
+        <h1 class="title tonic">Multi-Post Stories</h1>
+
```

Comment:
- [ ] Why are you not considered to be using other heading elements `h2` .... `h6` since you have already used `h1` in the header tag and it is recommended to use the `h1` element once in an HTML document. :blush: